### ISSUE TITLE
resize performance issues in IE11 (throttle)

### ISSUE THREAD

**Issue by haakemon** [bokuweb/re-resizable#248]
### Overview of the problem

I think adding a throttle to the mouseMove could improve performance. I also think the mousemove event listeners can wait to be attached until mousedown - but thats less important here.

I might be able to work on this myself, but not right now. I also would like to see if a PR would be accepted for this first.

I'm using re-resizable **version** 4.4.8
   

My **browser** is:   Internet Explorer 11
   
I am sure this issue is **not a duplicate**

### Description

When resizing containers in IE11, the experience is not smooth. The more complex the container, the more "lag" will be experienced. My use case involves a fairly complex container and it is barely usable in IE11.

### Steps to Reproduce

1. Create a slightly more complex container than a simple div to resize
2. Resize the div
3. The resize wil be "lagging" because of the repeated calculations runnning, and IE's crappy JS engine.

### Expected behavior

A smoother resize experience in IE

### Actual behavior

Lagging when resizing

**Comment by Dergash** [comment]
I have also encountered similiar perfomance issues in ie11, but for me it seems that it's more of a 'layout trashing' problem (https://gist.github.com/paulirish/5d52fb081b3570c81e3a) caused by complex DOM and heavy CSS styling, and not by JS. 
I've attached a screen with my ie11 perfomance session showing that every time I move a mouse while dragging, size and positioning computation (the green thingy) occured for the whole page from the root `<html>` tag.

![ie11-resizable-root-reflow](https://user-images.githubusercontent.com/13132449/43521384-3680bfd0-95c0-11e8-9dbd-1fa8cf1c4c7b.JPG)

I searched a bit and it seems there is a technique called 'layout boundaries' (http://wilsonpage.co.uk/introducing-layout-boundaries/) that allows to limit the scope of layout computation, which probably can help to smooth the ie11 resizing.
Could it be done or that is not the case here?
